### ISSUE TITLE
Extension Host crashes while formatting xml

### ISSUE THREAD

**Issue by Noderino** [CoenraadS/Bracket-Pair-Colorizer-2#38]
Extension host crashes while formatting xml documents with "Beautify" or "Xml Tools". A quick workaround to that is to add "xml" to excluded languages list.

VSCode: 1.29.0
aaron-bond.better-comments-2.0.3
andys8.jest-snippets-1.6.1
arjun.swagger-viewer-2.1.0
coenraads.bracket-pair-colorizer-2-0.0.23
dotjoshjohnson.xml-2.3.2
eamodio.gitlens-8.5.6
editorconfig.editorconfig-0.12.5
eg2.tslint-1.0.40
formulahendry.code-runner-0.9.5
hookyqr.beautify-1.4.7
joostlubach.js-index-0.2.1
ms-mssql.mssql-1.4.0
ms-vscode.autorest-2.0.1
ms-vscode.vs-keybindings-0.2.0
ms-vscode.vscode-typescript-tslint-plugin-0.0.5
msjsdiag.debugger-for-chrome-4.11.1
orta.vscode-jest-2.9.2
pflannery.vscode-versionlens-0.22.0
redhat.vscode-yaml-0.1.0
robertohuertasm.vscode-icons-7.28.0
shd101wyy.markdown-preview-enhanced-0.3.11
yzhang.markdown-all-in-one-1.7.0

Console Output:
```
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory

<--- Last few GCs --->

[4640:00000207450C31A0]    21543 ms: Mark-sweep 1761.2 (1824.0) -> 1761.2 (1824.0) MB, 1655.7 / 0.0 ms  allocation failure GC in old space requested
[4640:00000207450C31A0]    23200 ms: Mark-sweep 1761.2 (1824.0) -> 1761.1 (1792.5) MB, 1656.6 / 0.0 ms  last resort GC in old space requested
[4640:00000207450C31A0]    24961 ms: Mark-sweep 1761.1 (1792.5) -> 1761.1 (1792.5) MB, 1761.5 / 0.0 ms  last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 000000424162B0A1 <JSObject>
    1: tokenizeLine [C:\Users\Developer\.vscode\extensions\coenraads.bracket-pair-colorizer-2-0.0.23\out\src\documentDecoration.js:~224] [pc=00000148FFE639B5](this=000002B5412826F1 <DocumentDecoration map = 000000EC9DC34B89>,index=0)
    3: tokenizeDocument [C:\Users\Developer\.vscode\extensions\coenraads.bracket-pair-colorizer-2-0.0.23\out\src\documentDecoration.js:116] [bytecode=000002728E55...

```

**Comment by CoenraadS** [comment]
What a coincidence I just ran into this myself

**Comment by LebCit** [comment]
Thanks a lot man 🤣 
Sometimes, we are **too lazy** even too look at the extensions' options !

**Comment by CoenraadS** [comment]
Should be fixed in v0.0.24
